src/ec/packages/ec-component-text-input/text-input.html.twig

```diff
@@ -39,6 +44,7 @@
 {% if _invalid %}
   {% set _css_class = _css_class|merge(['ecl-text-input--invalid']) %}
 {% endif %}
+{% set _css_class = _css_class|merge(['ecl-text-area--' ~ _width ]) %}
```

Comment:
ok, this is the line where the class get messed up.. ;)

Reply:
`ecl-text-area--` replaced by `ecl-text-input--`